apps/golden-sample-app/src/app/app.module.ts

```diff
@@ -2,9 +2,9 @@ import { HttpClientModule, HTTP_INTERCEPTORS } from '@angular/common/http';
 import { APP_INITIALIZER, ErrorHandler, NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
-import { ACCESS_CONTROL_BASE_PATH } from '@backbase/data-ang/accesscontrol';
```

Comment:
Good job!
I was planning to create a ticket for removing data-ang but it seems you already did.